### PR TITLE
once support in addEventListener()

### PR DESCRIPTION
**Pull request by jeffposnick**
/web/updates article to accompany https://www.chromestatus.com/feature/5630331130478592, launching in Chrome 55.

### CODE REVIEW

File: src/content/en/updates/2016/10/addeventlistener-once.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+third parameter. Recent changes to the [method's
+definition](https://dom.spec.whatwg.org/#dom-eventtarget-addeventlistener) allow
+developers to provide additional options via a [configuration
+object](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Parameters),
+while remaining backward compatible when there's a boolean parameter or when an
+option is not specified.
+
+We're happy to announce that Chrome 55 adds support for the `once` option in
+that configuration object, alongside the `passive` ([implemented in Chrome
+51](https://developers.google.com/web/updates/2016/06/passive-event-listeners))
```
Comment (petele) [inline src/content/en/updates/2016/10/addeventlistener-once.md]: Don't include the domain, only use the absolute page.

File: src/content/en/updates/2016/10/addeventlistener-once.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+Note: The third parameter has historically been a boolean value, which defaulted
+to `false`, and which
+[controlled](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Parameters)
+whether the event listener was in [capturing or bubbling
+mode](http://stackoverflow.com/a/4616720/385997).
+
+## A more configurable addEventListener()
+
+The `addEventListener()`method has come a long way since the early days of the
```
Comment (petele) [inline src/content/en/updates/2016/10/addeventlistener-once.md]: missing space

File: src/content/en/updates/2016/10/addeventlistener-once.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    function setUpListeners() {
+      var data = ['one', 'two', '...etc.'];
+    
+      window.addEventListener('load', function() {
+        doSomethingWithSomeData(data);
+        // data is now part of the callback's scope.
+      });
+    }
+
+By default, the `'load'` event listener callback will remain in scope when it
```
Comment (petele) [inline src/content/en/updates/2016/10/addeventlistener-once.md]: load in double quotes?

**Comment by jeffposnick** [comment]
@petele, PTAL.

**Pull request merged**